src/main/java/com/metamx/emitter/core/Emitters.java

```diff
@@ -72,6 +72,10 @@ else if (props.getProperty(HTTP_EMITTER_PROP) != null) {
     httpMap.put("recipientBaseUrl", baseUrl);
     httpMap.put("flushMillis", Long.parseLong(props.getProperty("com.metamx.emitter.flushMillis", "60000")));
     httpMap.put("flushCount", Integer.parseInt(props.getProperty("com.metamx.emitter.flushCount", "300")));
+    /**
```

Comment:
:) If I am not wrong this class is only used for unit testing and this value should matter right ? If I give any other defaultValue for property `com.metamx.emitter.http.flushTimeOut` here apart from `HttpEmitterConfig.DEFAULT_FLUSH_TIME_OUT` the `testDefaults()` test will fail ?